uefi-test-runner/src/proto/mod.rs

```diff
@@ -22,12 +21,13 @@ pub fn test(image: Handle, st: &mut SystemTable<Boot>) {
     rng::test(bt);
     shell_params::test(bt);
     string::test(bt);
+    misc::test(image, bt);
 
     #[cfg(any(
-        target_arch = "x86",
-        target_arch = "x86_64",
-        target_arch = "arm",
-        target_arch = "aarch64"
+    target_arch = "x86",
```

Comment:
why did you change the indentation?

Reply:
It auto reformat by Clion/IDEA, sry.

Reply:
You can configure CLion to default to rustfmt.

![image](https://github.com/rust-osdev/uefi-rs/assets/5737016/88aac34b-2d40-4a0d-9042-4a352737b212)